third_party/brotli/c/dec/huffman.c

```diff
@@ -8,12 +8,16 @@
 
 #include "./huffman.h"
 
-#include <string.h>  /* memcpy, memset */
-
 #include "../common/constants.h"
 #include "../common/platform.h"
 #include <brotli/types.h>
 
+#if !defined(STARBOARD)
+#include <string.h>  /* memcpy, memset */
+#else
+#include "starboard/client_porting/poem/string_poem.h"
```

Comment:
@y4vor do we need string_poem anywhere still ? It only aliases `strdup` to Starboard currently, perhaps it's better to localize this where it's needed.

Reply:
If we don't use `strdup` we don't need it in this file.